packages/react-scripts/config/webpack.config.js

```diff
@@ -97,68 +97,6 @@ const sassModuleRegex = /\.module\.(scss|sass)$/;
 // *** Text-Em-All Web App
 const shouldDisableManifestGeneration = process.env.DISABLE_MANIFEST === 'true';
 const shouldDisableWebStatsGeneration = process.env.DISABLE_WEBSTATS === 'true';
-const shouldEnableDeadFileOutput = process.env.ENABLE_DEADFILE_OUTPUT;
-const entrypointTemplate = path.join(
-  paths.appSrc,
-  'entrypoint-script-template.js'
-);
-
-// Generates an HTML file with the <script> injected for every Text-Em-All Web App entry point.
-const makeHtmlPluginEntryForPage = (
-  entryPoint,
-  templatePath,
-  filename,
-  minifyOptions
-) =>
-  new HtmlWebpackPlugin(
-    Object.assign(
-      {},
-      {
-        inject: true,
-        chunks: [entryPoint],
-        template: templatePath,
-        filename,
-      },
-      minifyOptions
-    )
-  );
-
-// Generates a manifest plugin for the specified bundle/entrypoint
-//  to be written at the indicated path.
-const makeManifestPluginForBundle = entryPoint =>
-  new WebpackManifestPlugin({
-    fileName: `asset-manifest-${entryPoint}.json`,
-    publicPath: paths.publicUrlOrPath,
-    generate: (seed, files, entrypoints) => {
-      const manifestFiles = files.reduce((manifest, file) => {
-        manifest[file.name] = file.path;
-        return manifest;
-      }, seed);
-      const entryPointItem = entrypoints[entryPoint] || entrypoints.main;
-      const entrypointFiles = entryPointItem
-        ? entryPointItem.filter(fileName => !fileName.endsWith('.map'))
-        : [];
-
-      return {
-        files: manifestFiles,
-        entrypoints: entrypointFiles,
-      };
-    },
-  });
-
-// Generates a monolithic javascript file with the contents of all dependant chunks
-//  for every Text-Em-All Web App entry point. These are made available to external
-//  apps to load Text-Em-All Web App bundles - such as forms.
-const makeHtmlPluginEntryForBundle = (entryPoint, filename) =>
-  new HtmlWebpackPlugin({
-    inject: false,
-    chunks: [entryPoint],
-    template: entrypointTemplate,
-    filename: filename,
-    minify: false,
-    cache: false,
-  });
-// *** Call-Em-All Text-Em-All Web App
```

Comment:
May want to keep this line 161 to indicate end of our additions in this block. Just `// *** Text-Em-All Web App`